README.rst

```diff
@@ -23,6 +23,13 @@ The implementation is based on `PyTorch <https://pytorch.org>`_
 and `Lightning <https://www.pytorchlightning.ai>`_ for deep learning, and
 `Syne Tune <https://github.com/awslabs/syne-tune>`_ for hyperparameter optimization.
 
+Quick links
```

Comment:
i would add an installation section here as well. it's very common to have that. just installation followed by pip install renate. currently too hard to find.

Reply:
OK, I will add it.